scaladoc/src/dotty/tools/scaladoc/SocialLinks.scala

```diff
@@ -5,11 +5,15 @@ enum SocialLinks(val url: String, val className: String):
   case Twitter(tUrl: String) extends SocialLinks(tUrl, "twitter")
   case Gitter(gUrl: String) extends SocialLinks(gUrl, "gitter")
   case Discord(dUrl: String) extends SocialLinks(dUrl, "discord")
+  case Custom(cUrl: String, lightIcon: String, darkIcon: String) extends SocialLinks(cUrl, "custom")
 
 object SocialLinks:
+  val LowercaseNamePattern = "^[a-z]+$".r
+
   def parse(s: String): Either[String, SocialLinks] =
     val errorPrefix = s"Social links arg $s is invalid: "
     val splitted = s.split("::")
+
     splitted.head match {
```

Comment:
What do you think about maybe adding `.toLowerCase` here? I think it might be annoying if someone passes in `Custom` and it fails because it wants `custom`.

Reply:
Hello Chris, ho I hadn't thought of that, but it's much better. I'll test it to see if it's not a problem elsewhere.